tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -5289,661 +5289,51 @@ TEST (nnstreamer_capi_singleshot, invoke_dynamic_fail_n)
-  EXPECT_EQ (status, ML_ERROR_NONE);
-
-  /* Test Code */
-  status = ml_pipeline_element_get_property (NULL, "sync_mode", &ret_mode, NULL);
-  EXPECT_NE (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_element_get_property (elem_h, NULL, &ret_mode, NULL);
-  EXPECT_NE (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_element_get_property (elem_h, "wrong_mode", &ret_mode, NULL);
-  EXPECT_NE (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_element_get_property (elem_h, "sync_mode", &ret_mode,
-    "wrong_silent", &ret_silent, NULL);
-  EXPECT_NE (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_element_release_handle (elem_h);
-  EXPECT_EQ (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_destroy (handle);
-  EXPECT_EQ (status, ML_ERROR_NONE);
-
-  g_free(pipeline);
-}
-
-/**
- * @brief Test NNStreamer element-wise control functions
- * @detail run the ml_pipeline_element_get_property() api for valve & selector and check its error code.
- */
-TEST (nnstreamer_capi_element, get_property_04_n)
-{
-  ml_pipeline_h handle;
-  ml_pipeline_element_h valve_h;
-  ml_pipeline_element_h selector_h;
-  gboolean ret_valve_status;
-  gboolean ret_buffer_status;
-  guint ret_sync_mode;
-  gchar *pipeline;
-  int status;
-
-  /* valve & selector test */
-  pipeline = g_strdup("videotestsrc is-live=true ! videoconvert ! videoscale ! " \
-    "video/x-raw,format=RGBx,width=224,height=224,framerate=60/1 ! tensor_converter ! " \
-    "valve name=valvex ! input-selector name=is01 ! tensor_sink name=sinkx");
-
-  status = ml_pipeline_construct (pipeline, NULL, NULL, &handle);
-  EXPECT_EQ (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_element_get_handle (handle, "valvex", &valve_h);
-  EXPECT_EQ (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_element_get_handle (handle, "is01", &selector_h);
-  EXPECT_EQ (status, ML_ERROR_NONE);
-
-  /* Test Code */
-  status = ml_pipeline_element_get_property (NULL, "drop", &ret_valve_status, NULL);
-  EXPECT_NE (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_element_get_property (valve_h, NULL, &ret_valve_status, NULL);
-  EXPECT_NE (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_element_get_property (valve_h, "wrong_drop", &ret_valve_status, NULL);
-  EXPECT_NE (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_element_get_property (NULL, "cache-buffers", &ret_buffer_status,
-    "sync-mode", &ret_sync_mode, NULL);
-  EXPECT_NE (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_element_get_property (selector_h, "wrong-cache-buffers", &ret_buffer_status,
-    "sync-mode", &ret_sync_mode, NULL);
-  EXPECT_NE (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_element_release_handle (valve_h);
-  EXPECT_EQ (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_element_release_handle (selector_h);
-  EXPECT_EQ (status, ML_ERROR_NONE);
-
-  status = ml_pipeline_destroy (handle);
-  EXPECT_EQ (status, ML_ERROR_NONE);
+  error:
+    ml_pipeline_element_release_handle (elem_h);
+    ml_pipeline_destroy (handle);
```

Comment:
How about initializing `elem_h` and `handle` to `NULL`?

For example, I think the goto statement in line 5307 makes some problem due to uninitialized `elem_h`